coins/monero/src/wallet/mod.rs

```diff
@@ -218,4 +242,32 @@ impl Scanner {
       self.pair.view.deref() * spend,
     )
   }
+
+  /// Returns a featured address.
+  ///
+  /// if you created the scanner with `Some` value for `burning_bug` be aware that
```

Comment:
"A Scanner will fail to scan this address unless it is appropriate to the Scanner. If the Scanner was defined as not handling the burning bug, by passing `None` for `burning_bug` upon construction, this must be a guaranteed address in order to be scanned by it."